tsdb/points.go

```diff
@@ -52,6 +54,9 @@ type point struct {
 
 	// binary encoded field data
 	data []byte
+
+	// precision is set
+	precision string
```

Comment:
no, we don't need to.  I can chase the callstack up the chain and pass it through.  It felt a couple calls deep and I wasn't sure about passing it all they way down and see how it looks.